src/js/store/factory/Scale.ts

```diff
@@ -16,7 +16,8 @@ export type LyraScale = {
    */
   _origName: string;
   _domain: any[],
-  _range: any[]
+  _range: any[],
+  nice: any
```

Comment:
as afar as i know, no need to add this here. it's defined in the definitions of e.g. `ThresholdScale`, `QuantizeScale`, etc, which are part of `RangeScale`, and included when we use `& RangeScale` to sort of extend `RangeScale`.